app/controllers/api/databases.php

```diff
@@ -2535,20 +2535,19 @@ function updateAttribute(
             throw new Exception(Exception::COLLECTION_NOT_FOUND);
         }
 
-        $indexes = $collection->getAttribute('indexes');
-
-        // Search for index
-        $indexIndex = array_search($key, array_map(fn($idx) => $idx['key'], $indexes));
```

Comment:
Instead of doing getAttribute() and then searching through the result, we should probably be using `Document->find()`: https://github.com/utopia-php/database/blob/main/src/Database/Document.php#L280

Reply:
done this